### ISSUE TITLE
issue in translating day of week combination

### ISSUE THREAD

**Issue by a-dekker** [Salamek/cron-descriptor#6]
Hi Salamek,

I noticed the combination of individual weekdays and a range leads to syntax errors:
```python
>>> from cron_descriptor import get_description, ExpressionDescriptor
>>> print(get_description("38 20 * * 0,2,3-6"))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python2.7/site-packages/cron_descriptor/ExpressionDescriptor.py", line 568, in get_description
    return descripter.get_description(DescriptionTypeEnum.FULL)
  File "/usr/lib/python2.7/site-packages/cron_descriptor/ExpressionDescriptor.py", line 101, in get_description
    description = choices.get(description_type, self.get_seconds_description)()
  File "/usr/lib/python2.7/site-packages/cron_descriptor/ExpressionDescriptor.py", line 151, in get_full_description
    raise FormatException(description)
cron_descriptor.Exception.FormatException: An error occured when generating the expression description.  Check the cron expression syntax.
```
Just ranges or individual weekdays do work ok.

**Comment by Salamek** [comment]
@a-dekker fixed in release 1.2.5, pip updated

**Comment by a-dekker** [comment]
No longer getting the error, thanks for the fix!
